lib/unique_elements.rb

```diff
@@ -2,5 +2,15 @@
 # combination of the above), and returns an array of the elements that appear once and only
 # once.
 def find_unique_elements(arr)
-  
+    #select each element of the array and compare each one to the rest of the array
+    arr.each_index do |idx|
+    #compare each element within the array
+        if arr[idx] == arr[idx+1]
+            arr.delete_at(idx)
+            #delete the first matching element
+            arr.delete_at(idx+1)
```

Comment:
This is the question that I asked you about and you helped me get to the first solution I submitted. I changed .delete_at to .delete but I'm not sure if that's what you're meaning by "a small change in how you are deleting values". I can try using a different method if that's what you are meaning. Please let me know what you're looking for and I will do my best to change it to what you'd like to see.